decidim-assemblies/spec/commands/admin/destroy_area_with_assemblies_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim::Admin
+  describe DestroyArea do
+    subject { described_class.new(area, user) }
+
+    let(:organization) { create :organization }
+    let(:user) { create :user, :admin, :confirmed, organization: organization }
+    let(:area) { create :area, organization: organization }
+
+    context "when an assembly associated to a given area exist" do
```

Comment:
This test is redundant, it's already defined in `decidim-admin/spec/commands/decidim/admin/destroy_area_spec.rb`.

Is there any reason to duplicate it?

Reply:
I'm removing them from here now.
They shouldn't be here as they couple this module with participatory texts and assemblies, so I moved the tests there, but forgot the removal from here.